content/en/docs/tasks/security/authorization/authz-custom/index.md

```diff
@@ -0,0 +1,235 @@
+---
+title: External authorization with custom action
+description: Shows how to integrate and delegate access control to an external authorization system.
+weight: 35
+keywords: [security,access-control,rbac,authorization,custom, opa, oauth, oauth2-proxy]
+owner: istio/wg-security-maintainers
+test: no
+---
+
+This task shows you how to set up Istio authorization policy with a [`CUSTOM` action](https://github.com/istio/api/blob/328c3a37131887008ddc8ed469f5296e57c27946/security/v1beta1/authorization_policy.proto#L296)
```

Comment:
Hmm, pointing to a specific line number in the .proto file is very fargile. It will be off as soon as any change is made to the file. I didn't realize that the CUSTOM value is currently hidden from docs.

Maybe, instead, it would be better to say something like this:

```suggestion
This task shows you how to set up an Istio authorization policy using a new experimental value for the [action field](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action), `CUSTOM`,
```